addons/hr_holidays/models/hr_leave.py

```diff
@@ -749,17 +749,16 @@ def _check_approval_update(self, state):
                 # use ir.rule based first access check: department, members, ... (see security.xml)
                 holiday.check_access_rule('write')
 
-            if holiday.employee_id == current_employee and not is_manager:
-                raise UserError(_('Only a Leave Manager can approve its own requests.'))
+            if holiday.employee_id == current_employee and not is_officer:
```

Comment:
This was really intended to be limited to managers as validating your own requests is quite powerful.
I don't know if we should allow holiday users to do this

Reply:
What do you mean by "holiday users"?

Reply:
hr_holidays.group_hr_holidays_user

Reply:
Yup, that's the xmlid for the Holidays Officer, so it's ok